persistent-postgresql/Database/Persist/Postgresql.hs

```diff
@@ -186,7 +186,7 @@ createPostgresqlPool = createPostgresqlPoolModified (const $ return ())
 --
 -- @since 2.1.3
 createPostgresqlPoolModified
-    :: (MonadUnliftIO m, MonadLogger m)
+    :: (MonadUnliftIO m, MonadLoggerIO m)
```

Comment:
Changing the signature of these functions is a breaking change and would require a major version bump.